doc/riscv_trace.rst

```diff
@@ -0,0 +1,70 @@
+.. _riscv_trace:                                                                                         
+
+
+RISC-V Trace Interface
+============================
+
+Ibex implements an instruction trace interface compatible with the `RISC-V Trace Specification, version 1.0, as released on March 20, 2020 <https://github.com/riscv/riscv-trace-spec/>`_. 
+This interface exports needed internal signals as ibex_core outputs to be used to encode the trace packets to be logged.
+
+Signals implemented until now are shown in the table below:
+
+
+Trace Signals
+-------------
+
++--------------------+---------------------+-------------------------------------------------------------+
+|   Signal Name      |   Number of bits    |                         Description                         |
```

Comment:
Remove all unnecessary whitespace around these fields to make them less wide and the source more readable.

"Number of bits" can be shortened to "Bits"